neo4j/api.py

```diff
@@ -80,13 +80,42 @@ def custom_auth(principal, credentials, realm, scheme, **parameters):
 class Bookmark:
 
     def __init__(self, value):
```

Comment:
Could the signature be `def __init__(self, *values)`? This would be compatible, but allow multiple to be supplied on construction.